pkg/apis/policy/v1alpha1/types.go

```diff
@@ -33,9 +33,10 @@ type Rule struct {
 
 // ResourceDescription describes the resource to which the PolicyRule will be applied.
 type ResourceDescription struct {
-	Kinds    []string              `json:"kinds"`
-	Name     *string               `json:"name"`
-	Selector *metav1.LabelSelector `json:"selector"`
+	Kinds     []string              `json:"kinds"`
+	Name      *string               `json:"name"`
+	Namespace *string               `json:"namespace,omitempty"`
+	Selector  *metav1.LabelSelector `json:"selector"`
```

Comment:
Do we need to change crd spec in the install.yaml accordingly?

Reply:
True, updated